src/autora/experimentalist/sampler/random_sampler.py

```diff
@@ -2,7 +2,7 @@
 from typing import Iterable, Sequence, Union
 
 
-def random_sampler(conditions: Union[Iterable, Sequence], n: int):
+def random_sample(conditions: Union[Iterable, Sequence], n: int = 1):
```

Comment:
this is good, but I think we should also add an "alias" for `random_sampler` so that old code doesn't break. We can put a deprecated warning around it so that if anyone uses it, they get pointed to the new name.

Reply:
I love that! Didn't think about it and was very concerned about breaking the code. Thanks!